eng/pipelines/templates/common.yml

```diff
@@ -178,6 +178,7 @@ extends:
           dependsOn:
           - Initialize
           - SignAndPack
+          - SignAndPackVSIX
```

Comment:
@tmeschter how are you doing this for big-box VS Marketplace?  Should we be adding a pipeline for the VS VSIX here too?

Reply:
For big-box, we should just produce additional artifacts in the existing steps and scripts.